tools/cythonize.py

```diff
@@ -55,9 +55,12 @@ def process_pyx(fromfile, tofile):
     try:
         from Cython.Compiler.Version import version as cython_version
         from distutils.version import LooseVersion
-        if LooseVersion(cython_version) < LooseVersion('0.19'):
-            raise Exception('Building %s requires Cython >= 0.19' % VENDOR)
-
+        if sys.version_info[:2] < (3, 7):
+            if LooseVersion(cython_version) < LooseVersion('0.19'):
+                raise Exception('Building %s requires Cython >= 0.19' % VENDOR)
```

Comment:
I actually think this is out of date also. Python 3.7 isn't the first to require a newer Cython, IIRC, we also had some problems with Python 3.6. I really don't know what required for the different Python releases. @rgommers Ideas?

Reply:
I'd just be tempted to just require >= 0.28.2, it is on pip and should serve for the next 18 months.

Reply:
Are you asking for me to add the `.2` to the python 3.7 condition. or to remove the 0.19 condition entirely?

Reply:
Remove entirely.